Makefile

```diff
@@ -1,11 +1,30 @@
-.PHONY: test unit_tests integration_tests
+DOT := $(shell command -v dot 2> /dev/null)
+
+
 
 build: install
 
 install:
 	pipenv install --dev
 
-test:
+test: unit_tests component_tests
 	pipenv check
 	pipenv run flake8 .
-	pipenv run pytest test/
+
+unit_tests:
+	pipenv run pytest test/unit/
+
+component_tests: docker_build
+	docker-compose up -d ;
+	./test/component/setup_pubsub.sh
+	pipenv run pytest test/component/
+
+up: docker_build
+	docker-compose up -d;
+	./setup_pubsub.sh
+
+down:
+	docker-compose down
+
+docker_build:
+	docker build -t census-rm-pubsub:latest .
```

Comment:
Should perhaps be the same image name as we use in census-rm-docker-dev, `eu.gcr.io/census-ci/rm/census-rm-pubsub` to allow us to easily run docker dev with local changes?

Reply:
changed, though this will be removed in next version